homeassistant/components/calendar/__init__.py

```diff
@@ -42,6 +46,14 @@ def async_setup(hass, config):
 DEFAULT_CONF_OFFSET = '!!'
 
 
+def get_date(date):
```

Comment:
This is a weird helper. It assumes it's handling a dictionary containing an RFC 5545 events.

I'm also confused that we're returning it as local? We are not rendering anything here, we should send it to the frontend as UTC time. The frontend can then render it how it wants (browser local time, instance local time)

Reply:
All events sent by the endpoint `api/calendar/calendar_ip` are UTC (ie `2018-06-15T03:59:59+00:00`)

Example: The todoist component returns an event with the due date: `2018-06-15T03:59:59+00:00`
The UI displays correctly this event as 2018-06-14T23:59:59-04:00 in my local time (Montréal)

Do you want I rename this function ?

Reply:
We should not return localized dates. We should return everything always as UTC.